src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -69,14 +66,6 @@ const GiftPopover = (
   </Popover>
 );
 
-const RadioPopover: React.FC<RadioModalPropsType> = (props) => (
-  <Popover id="radio-popover">
-    <Popover.Content>
-      <RadioModal {...props} />
-    </Popover.Content>
-  </Popover>
-);
-
```

Comment:
Not sure why we removed this in favour of inlining it?

Reply:
For whatever reason, it didnt receive the correct props it needed to be rendered connected to the correct OverlayTrigger component.

With it like this, it rendered randomly on the top left of the screen instead of bellow the radio button on the navbar.

The perfect solution should and, hopefully soon, will be to pull that code out and make it a standalone component

Reply:
No worries